### ISSUE TITLE
Doesn't work at JetBrains/IntelliJ terminal

### ISSUE THREAD

**Issue by nebiros** [kabinspace/AstroVim#100]
I made `nvim` my default `$EDITOR`, but, AstroVim doesn't work at the `JetBrains/IntelliJ` terminal, for some reason cannot render it properly:

<img width="1035" alt="Screen Shot 2022-02-25 at 9 53 57 AM" src="https://user-images.githubusercontent.com/97343/155763697-1142d173-b44d-48de-89f7-67cb04ffec3e.png">

**Comment by kabinspace** [comment]
@nebiros Are you trying to use AstroVIm inside JetBrains integrated terminal?

**Comment by bryant-the-coder** [comment]
why r u using astrovim in jetbrains?

**Comment by nebiros** [comment]
> @nebiros Are you trying to use AstroVIm inside JetBrains integrated terminal?

yes, inside the JetBrains/IntelliJ terminal, why?, cause I set my $EDITOR env var to nvim, 😅

**Comment by kabinspace** [comment]
@nebiros Does AstroVim work in your external terminal? except for JetBrains

**Comment by nebiros** [comment]
> @nebiros Does AstroVim work in your external terminal? except for JetBrains

exactly, in iterm2 at macos, which is the terminal I use, it works ok, but not inside the JetBrains/IntelliJ terminal

**Comment by kabinspace** [comment]
@nebiros Did neovim work in your JetBrains terminal before you installed AstroVim or not?

**Comment by nebiros** [comment]
> @nebiros Did neovim work in your JetBrains terminal before you installed AstroVim or not?

honestly, I don't know, first time using neovim cause AstroVim, before I was using vim/macvim
